dropshot/src/handler.rs

```diff
@@ -132,6 +131,10 @@ impl RequestInfo {
         &self.headers
     }
 
+    pub fn remote_addr(&self) -> &std::net::SocketAddr {
+        &self.remote_addr
+    }
```

Comment:
```suggestion
    pub fn remote_addr(&self) -> std::net::SocketAddr {
        self.remote_addr
    }
```

`SocketAddr` returns `Copy` so I think it's worthwhile to just return it directly rather than a reference.  That'll also be consistent with [`local_addr()` on the Server](https://docs.rs/dropshot/latest/dropshot/struct.HttpServer.html#method.local_addr).